src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -129,6 +130,53 @@ public void testGetTableData() throws SQLException {
         assertArrayEquals(expected, actual);
     }
 
+    @Test
+    public void testDataSetToString() {
+        DataSet dataSet = createDataSet(new String[]{"1", "name1", "25"});
+        assertEquals("'1','name1','25'", dataSet.toString());
+    }
+
+    @Test
+    public void testInsertWhenSuccess() throws SQLException {
```

Comment:
Use When to describe conditions instead of result

Reply:
Thanks for the advice, I rename the method